src/controllers/productController.ts

```diff
@@ -99,7 +96,7 @@ async function destroyProduct(req: Request, res: Response, next: NextFunction) {
   }
 }
 
-export const indexProductFuncs = [indexProduct, getMultipleIdFormatting];
+export const indexProductFuncs = [indexProduct, showProduct];
```

Comment:
Lets not mix the individual operation calls before it gets too confusing. `showProduct` should only be present in `showProductFuncs`. With the above modification, can just do `export const indexProductFuncs = [indexProduct];`

Reply:
Done